lib/ListActions/ListActions.js

```diff
@@ -677,6 +686,10 @@ var ListActions = module.exports = kind(
 	stackMeUp: function() {
 		var optionGroup, i;
 
+		if (this.standardHeight) {
```

Comment:
@aarontam  Thanks for your comment. You're right. It's a sort of repetitive codes necessary to be moved to somewhere. Like you said, {{beforeOpenDrawer}} is the proper place so I updated PR.